packages/amplify-codegen-appsync-model-plugin/src/visitors/appsync-typescript-visitor.ts

```diff
@@ -87,15 +87,15 @@ export class AppSyncModelTypeScriptVisitor<
     modelObj.fields.forEach((field: CodeGenField) => {
       modelDeclarations.addProperty(this.getFieldName(field), this.getNativeType(field), undefined, 'DEFAULT', {
         readonly: true,
-        optional: field.isNullable,
+        optional: field.isList ? field.isListNullable : field.isNullable,
       });
     });
     let ownerAuthRules = getOwnerAuthRules(modelObj);
     if (ownerAuthRules !== undefined) {
       for (let rule of ownerAuthRules) {
         let ownerFieldName = getOwnerFieldName(rule);
         if (ownerFieldName !== undefined && !hasOwnerField(modelObj.fields, ownerFieldName)) {
-          modelDeclarations.addProperty(ownerFieldName, "String", undefined, 'DEFAULT', {
+          modelDeclarations.addProperty(ownerFieldName, 'String', undefined, 'DEFAULT', {
```

Comment:
I think this is where the merging conflict is. I removed this part in my previous merged PR.

Reply:
@AaronZyLee , I did not notice the merge conflict... My editor must've refactored to single quotes. I am pushing the merge that resolves the conflict now.